packages/certs-v5/test/commands/certs/auto/enable.js

```diff
@@ -28,14 +28,19 @@ describe('heroku certs:auto:enable', function () {
       { 'kind': 'custom', 'hostname': 'foo.example.org', 'cname': 'foo.example.org.herokudns.com' }
     ])
 
-    return certs.run({ app: 'example' }).then(function () {
-      expect(cli.stderr).to.equal('Enabling Automatic Certificate Management... starting. See status with heroku certs:auto or wait until active with heroku certs:auto:wait\n')
-      expect(cli.stdout).to.equal(
-        `=== Your certificate will now be managed by Heroku.  Check the status by running heroku certs:auto.
+    return certs.run({
+        app: 'example',
+        args: ['--wait'],
```

Comment:
Why is this arg required in all tests?

Reply:
Mmm maybe it's not needed and it's only a copy-paste leftover, I'll check it.

Reply:
You were right, that arg wasn't needed , although the flag it's needed on the tests